content/blog/2014/09/2014-09-11-dhcpcd-cve-2014-6060/index.md

```diff
@@ -12,7 +12,7 @@ url = "/2014/09/11/dhcpcd-cve-2014-6060/"
 All Evolve OS users are advised to update their systems as soon as possible. A vulnerability exists in dhcpcd which allows a denial of service. We have backported the 
 relevant fix to our version of dhcpcd to ensure minimal disruption whilst maintaining security,
 
-[A description from the National Vulnerability Database:](http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2014-6060)
```

Comment:
Also this CVE link still works (with redirections, but that's not an issue for an old post)